openapi/api_trial_system.template.yaml

```diff
@@ -0,0 +1,133 @@
+swagger: "2.0"
+info:
+  version: 0.0.1
+  title: Trial System API
+host: localhost
+basePath: /trials/api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+securityDefinitions:
+  Bearer:
+    type: apiKey
+    name: Authorization
+    in: header
+paths:
+  "/subscriptions/{trialId}":
```

Comment:
**note**: Consider changing to `/trials/{trialId}/subscriptions`, making it more "RESTful-like" (WDYT @AleDore @kin0992 ?)

Reply:
absolutely :)

Reply:
Done ✅